src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,302 @@
+            if (currentFocus < 0) currentFocus = (x.length - 1);
+
+            x[currentFocus].classList.add("autocomplete-active");
+        }
+        function removeActive(x) {
+
+            for (var i = 0; i < x.length; i++) {
+                x[i].classList.remove("autocomplete-active");
+            }
+        }
+        function closeAllLists(elmnt) {
+
+            var x = document.getElementsByClassName("autocomplete-items");
+            for (var i = 0; i < x.length; i++) {
+                if (elmnt != x[i] && elmnt != inp) {
+                    x[i].parentNode.removeChild(x[i]);
+                }
+            }
+        }
+
+        document.addEventListener("click", function (e) {
+            closeAllLists(e.target);
+        });
+    }
+
+
+render() {
+
+
+    return (
+        <div className="todoListMain">
+            <div className='header'>
+                <form autocomplete="off" action="/action_page.php" onSubmit={this.addItem} >
+                    <input ref={(a) => this._inputElement = a}
+                        placeholder="Enter ToDo item">
+                    </input>
+                    <button type="submit" >Add ToDo</button>
+                    <button onClick={this.deleteItem}>Clear Completed</button>
+                    <div className="pomodoro-wrapper">
+                        <div id="pomodoro-app">
+                            <div id="container">
+                                <div id="timer">
+                                    <div id="time">
+                                        <span id="minutes">25</span>
+                                        <span id="colon">:</span>
+                                        <span id="seconds">00</span>
+                                    </div>
+                                    <div id="filler"></div>
+                                </div>
+
+                                <div id="buttons">
+                                    <button id="work">Work</button>
+                                    <button id="shortBreak">Short Break</button>
+                                    <button id="longBreak">Long Break</button>
+                                    <button id="stop">Stop</button>
+                                </div>
+                            </div>
+                        </div>
+                        <img src="http://cdn.shopify.com/s/files/1/1940/0783/products/Tomato_timer_01_1024x1024.png?v=1523492026" alt="tomato-pic"></img>
+
+
+                    </div>
+                    <div className="autocomplete" >
+                        <input id="myInput" type="text" name="myTodo" placeholder="Search Todo">
+                        </input >
+
+                    </div>
+
+                </form>
+            </div>
+            <Todo entries={this.state.items}
+                delete={this.deleteItem}
+                markCompleted={this.markCompleted} />
+
+        </div>
+    );
+}
+}
+
```

Comment:
This has gotten to be a pretty large file. Think of what things you could break out into smaller components to make this thing a little more readable.